Safari Work/safari-specific-bugs.md

```diff
@@ -1,79 +1,79 @@
 # Safari-Specific Bugs for Discord 
-## List is outdated and Safari keyboard navigation bugs are put on hold, don't interact with this 
 ✅ = 1 Approval  
 ❌ = 1 Denial  
 
 ~~12271 ✅✅✅~~  
 ~~14043 ✅✅✅~~  
-14165  
+~~14165 ❌❌❌~~
 ~~15191 ❌❌❌~~  
 ~~15247 ✅✅✅~~    
-15250 ✅  
+15250 ✅ (?) (This report does not exist) 
 ~~15252 ✅✅✅~~  
 ~~15316 ✅✅✅~~  
 ~~15335 ✅✅✅~~  
-15385  
+~~15385❌❌❌~~
 ~~15413 ❌❌❌~~  
 ~~15422 ❌❌❌~~   
-15424  
-15526 ✅  
+15424  ✅❌
+~~15526❌❌❌~~
 15527 ❌   
-15528   
-15529 ❌   
+~~15528 ✅✅✅~~
+~~15529 ❌❌❌ ~~
 ~~15530 ✅✅✅~~     
-15531 ✅   
-15532    
-15534 ✅   
-15535    
+~~15531 ✅✅✅~~ 
+15532 ✅   
+~~15534 ✅✅✅~~
+15535 ✅
 ~~15536 ✅✅✅~~      
 15538 ✅   
-15539   
-15547   
-15548   
-15551   
-15555   
-15556 ✅   
-15557 ✅  
+~~15539 ✅✅✅~~ (Not safari specific)
+~~15547 ❌❌❌~~
+15548 ✅
+~~15551 ✅✅✅~~   
+15555 ✅ 
+~~15556 ✅✅✅~~ 
+~~15557 ✅✅✅~~
 15558   
-15559 ✅✅      
+~~15559 ✅✅✅~~     
 15560 ✅  
-15562  
-15565 ❌   
-15566  
-15567 ✅   
-15568   
+~~15562 ✅✅✅~~  
+~~15565 ❌❌❌~~
+~~15566 ❌❌❌~~  
+15567 ✅✅✅   
+~~15568 ❌❌❌~~ 
 ~~15569 ✅✅✅~~   
-15570   
-15571   
-15573   
-15574 ❌ (Something to do with pop ups but test anyways)   
-15578  
-15582   
-15585 ❌   
-15586  
-15592  
-15599  
-15601  
-15748 ✅✅       
-15882    
-15823 ✅    
-15825   
-15826 ✅✅   
-15827 ✅   
-15828  
-15829   
+~~15570 ✅✅✅~~
```

Comment:
This report is still in queue with 0 interactions. I don't have time do fully spot check this list, can you please re-check these?

Reply:
I will when I will get back to my PC

Reply:
@wakfi Just did some updating it should be better now. But still need check.